Sources/PostgreSQL/Connection/PostgreSQLConnection+ServerAddress.swift

```diff
@@ -11,6 +11,16 @@ extension PostgreSQLConnection {
             return .unixSocket(path: "/tmp/.s.PGSQL.5432")
         }
         
+        /// TCP or Unix socket PostgreSQL address depending on the host format.
+        public static func tcpOrSocket(hostname: String, port: Int) -> ServerAddress {
```

Comment:
Could we just make this part of what the existing `tcp` method does? I can't think of any hostnames that start with `/`, so there shouldn't be a problem of ambiguity.

Reply:
It's done.